Mage/src/main/java/mage/choices/FaceVillainousChoice.java

```diff
@@ -0,0 +1,50 @@
+package mage.choices;
+
+import mage.abilities.Ability;
+import mage.constants.Outcome;
+import mage.game.Game;
+import mage.game.events.GameEvent;
+import mage.players.Player;
+
+/**
+ * @author TheElk801
+ */
+public class FaceVillainousChoice {
```

Comment:
It's a little strange how these classes don't extend Effect or anything, but it seems fine and handles the text gen well.

Reply:
yeah it's the same for `ChooseFriendsAndFoes`, there's just nothing that needs to actually be extended